card/mod/01_history/set/all/history.rb

```diff
@@ -317,14 +348,17 @@ def fold_or_unfold_link args
       toggled_view = :act_expanded
     end
     link_to '', args.merge(view: toggled_view),
-              class: "slotter revision-#{args[:act_id]} #{ args[:act_view]==:expanded ? "arrow-down" : "arrow-right"}",
+              class: "slotter revision-#{args[:act_id]} #{ args[:act_view] == :expanded ? 'arrow-down' : 'arrow-right'}",
```

Comment:
Line is too long. [121/80]<br>Align the parameters of a method call if they span more than one line.<br>Space inside string interpolation detected.